compiler/rustc_borrowck/src/diagnostics/mutability_errors.rs

```diff
@@ -864,14 +868,9 @@ impl<'a, 'tcx> MirBorrowckCtxt<'a, 'tcx> {
             } else {
                 bug!("not an upvar")
             };
-            err.span_label(
-                *span,
-                format!(
-                    "calling `{}` requires mutable binding due to {}",
-                    self.describe_place(the_place_err).unwrap(),
-                    reason
-                ),
-            );
+            let place = self.describe_place(the_place_err).unwrap();
```

Comment:
We'll need to check if `describe_place` includes any English text because if it does then this won't be translatable.

Reply:
[I belive I've digged deep enough](https://github.com/rust-lang/rust/pull/101276#discussion_r965561753)